app/timetables/models.py

```diff
@@ -249,3 +249,13 @@ class Serving(TimestampMixin):
 
     def __str__(self):
         return '{} served on {}'.format(self.menu_item, self.date_served)
+
+
+class TimetableAutoUpdate(models.Model):
+    """Model to track calls to the timetable auto-update script."""
+
+    timetable = models.ForeignKey(Timetable, on_delete=models.CASCADE)
```

Comment:
@andela-kanyanwu: I think a OneToOneField should be used here. Or would it be correct to have multiple entries of same timetable reference in here?